src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function(){
+    $("#button_submit").on('click', changeStudnetInformationConfirmation);
```

Comment:
Changes to email are handled and cascaded to feedback responses, so there are no deletions for that case.